src/plugins/intel_npu/src/plugin/npuw/partitioning/online/utils/utils.cpp

```diff
@@ -44,33 +44,59 @@ std::string ov::npuw::online::util::repeated_id(const std::shared_ptr<Repeated>&
     return ss.str();
 }
 
-std::optional<ov::npuw::online::Avoid> ov::npuw::online::util::parseAvoid(const std::string& s) {
+std::tuple<ov::npuw::online::PatternType, std::string, std::string> ov::npuw::online::util::parse(
+    const std::string& s) {
     auto pos_col = s.find(':');
     auto pos_sl = s.find('/');
 
     if (pos_col == std::string::npos || pos_sl == std::string::npos) {
-        LOG_WARN("Incorect pattern in OPENVINO_NPUW_AVOID: "
-                 << s << ". Please, separate a device with / and pattern type with :, e.g. Op:Select/NPU,P:RMSNorm/NPU."
-                 << " Avoid rule " << s << " is ommited!");
-        return std::nullopt;
+        LOG_WARN("Incorrect pattern in OPENVINO_NPUW_AVOID or OPENVINO_NPUW_ISOLATE: "
+                 << s << ". Please, separate a device or tag with / and pattern type with :."
+                 << " Rule " << s << " is ommited!");
+        return {};
     }
 
     auto type = s.substr(0, pos_col);
     auto pattern = s.substr(pos_col + 1, pos_sl - pos_col - 1);
-    auto device = s.substr(pos_sl + 1, s.size() - pos_sl - 1);
+    auto device_or_tag = s.substr(pos_sl + 1, s.size() - pos_sl - 1);
 
     if (type != "Op" && type != "P") {
-        LOG_WARN("Incorect pattern type in OPENVINO_NPUW_AVOID: "
+        LOG_WARN("Incorrect pattern type in OPENVINO_NPUW_AVOID or OPENVINO_NPUW_ISOLATE: "
                  << type << ". Please, use either Op for operation or P for pattern."
-                 << " E.g. Op:Select/NPU,P:RMSNorm/NPU."
-                 << " Avoid rule " << s << " is ommited!");
+                 << " Rule " << s << " is ommited!");
+        return {};
+    }
+
+    auto pattern_type = type == "Op" ? PatternType::OP : PatternType::PATTERN;
```

Comment:
Do you check that type can be _ONLY_ "Op" or "P"? What if I put "X:Something/Somewhere" ?

Reply:
Well it will do nothing if it's not reflected in the model. I don't think we should do anything with that

Reply:
No, because there is an assert in parser that it's either P or Op

Reply:
The assert was here before, but due to unification and code reduction I've moved it in utils.cpp